tests/resources/maps.yaml

```diff
@@ -0,0 +1,18 @@
+
+section:
+    map:
+    - '{key1:value1'
+    - key2:value2
+    - key2:value3}
+    map_spaces:
+    - '{ key1 : value1'
+    - 'key2 : value2'
+    - 'key2 : value3 }'
+    map_quotes:
+    - '{"key1":"value1"'
+    - '"key2":"value2"'
+    - '"key2":"value3"}'
+    map_quotes_spaces:
+    - '{ "key1" : "value1"'
+    - '"key2" : "value2"'
+    - '"key2" : "value3" }'
```

Comment:
This map configuration is just an example which is currently not supported, right? Can't find tests for this.